packages/app/src/components/DBTracePanel.tsx

```diff
@@ -56,7 +57,7 @@ export default function DBTracePanel({
 
   const [traceRowWhere, setTraceRowWhere] = useQueryState(
     'traceRowWhere',
```

Comment:
not a huge fan of overloading this query param, esp if the rowWhere is for a log but put in the "traceRowWhere" - the naming wouldn't make sense.

I think it might be better to introduce another query param for `logRowWhere`? We could rename this query param but it'd technically be a breaking change (technically this schema change is already a breaking change, URLs are supposed to be stable in our app).

Reply:
Yeah, I agree with you we should not overload `traceRowWhere`, it should only return trace relate param, however I think it is a little too overcomplicate to make two queryState for returning focus param.

I think I will change the name to `spanRowWhere`, it makes more sense since span can be log or trace.

Reply:
Ah, thanks for the explain.
And for another approach, I think it is a good idea, probably can create a hook function for this. I will take a look. But for now I am just changing the name to `event` to make sure this PR is in scope.